roles/openstack-stack/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: create HOT stack template prefix
+  register: stack_template_pre
+  tempfile:
+    state: directory
+    prefix: casl-ansible
```

Comment:
Agreed - I think we should aim to merge as much as-is to have a working implementation, then do smaller follow-up PRs to do clean-up / renaming.